src/Connection.php

```diff
@@ -65,9 +68,10 @@ public function __construct(array $config)
 
         // Create the connection
         $this->connection = $this->createConnection($dsn, $config, $options);
+        $this->database = $this->getDefaultDatabaseName($dsn, $config);
```

Comment:
Is this dynamically assigning a class property, or is this declared on BaseConnection?

Either way, this appears to be the first time it's being used.

Reply:
The property is defined [in the parent class](https://github.com/laravel/framework/blob/45f0347e4baaab341fbab27031026b60ff3e9ffb/src/Illuminate/Database/Connection.php#L55), it was not set previously.